configuration/configuration.go

```diff
@@ -57,6 +57,9 @@ type Configuration struct {
 			// contain the private portion for the file specified in
 			// Certificate.
 			Key string `yaml:"key,omitempty"`
+
+			// Specifies the CA cert for client authentication
+			CACert string `yaml:"ca,omitempty"`
```

Comment:
Should this be called `ClientCA` to match the tls configuration struct? Change the yaml to "clientca", as well.

Should it take a list to allow more than one CA?